tokio-quiche/src/quic/connection/mod.rs

```diff
@@ -196,7 +196,16 @@ pub struct Incoming {
     pub buf: PooledBuf,
     /// If set, then `buf` is a GRO buffer containing multiple packets.
     /// Each individual packet has a size of `gso` (except for the last one).
-    pub gro: Option<u16>,
+    pub gro: Option<i32>,
+    /// [SO_MARK] control message value received from the socket.
+    ///
+    /// This will only be populated for the first packet on a given connection.
+    /// This will always be `None` after the connection has been spawned as
+    /// the message is `take()`d before spawning.
```

Comment:
> This will only be populated for the first packet on a given connection.

Do we need a test for this case. The current test checks that we have this stat on the first packet.